packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -491,6 +520,34 @@ export function SelectPanel({
               </Box>
             ) : null}
           </Box>
+          {notice &&
+            (enabled ? (
+              <div aria-live="polite" data-variant={notice.variant} className={classes.Notice}>
+                {iconForNoticeVariant[notice.variant]}
+                <div>{notice.text}</div>
+              </div>
+            ) : (
+              <Box
+                aria-live="polite"
+                data-variant={notice.variant}
+                sx={{
+                  display: 'flex',
+                  gap: 2,
+                  paddingX: 3,
+                  paddingY: '12px',
+                  marginTop: 1,
+                  borderTop: '1px solid',
+                  borderBottom: '1px solid',
+                  fontSize: 0,
+                  a: {color: 'inherit', textDecoration: 'underline'},
+                  ...stylesForNoticeVariant[notice.variant],
+                }}
+                className={enabled ? classes.Notice : undefined}
+              >
+                {iconForNoticeVariant[notice.variant]}
+                <Box>{notice.text}</Box>
+              </Box>
+            ))}
```

Comment:
For new code, we're implementing with css modules only, see: https://github.com/primer/react/pull/5761#discussion_r2001481159)

Reply:
Oh, wonderful